src/instructlab/configuration.py

```diff
@@ -453,6 +455,7 @@ class _generate(BaseModel):
         description="Directory where generated datasets are stored.",
     )
     # TODO: remove this? It's not used anywhere, was removed by 19b9f4794f79ef81578c00c901bac3ee9db8c046
+    # related issue: https://github.com/instructlab/instructlab/issues/2261
```

Comment:
nit: not a huge fan of mentioning issues in code

Reply:
Valid opinion, but this is done elsewhere in the codebase at this time - going to keep it for now